orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -56,7 +56,7 @@ errorExit() {
 kinit_as_hdfs() {
   # Use a new Kerberos credential cache, to keep from clobbering the default.
   export KRB5CCNAME=/tmp/krb5cc_cloudbreak_$EUID
-  HDFS_KEYTAB=/run/cloudera-scm-agent/process/$(ls -t /run/cloudera-scm-agent/process/ | grep hdfs-NAMENODE$ | head -n 1)/hdfs.keytab
+  HDFS_KEYTAB=/run/cloudera-scm-agent/process/$(ls -t /run/cloudera-scm-agent/process/ | grep "hdfs-\(JOURNALNODE\|DATANODE\|NAMENODE\)$" | head -n 1)/hdfs.keytab
```

Comment:
This will work as long one of the service listed below are running in the gateway node where salt-master is running.

1. JOURNALNODE
2. DATANODE
3. NAMENODE

Extend this to use keytab's for Solr and Hbase services as well if the HDFS keytab's are not found.

Later we could update the logic in CB to run this command on a host where Solr/Hbase services are running instead of always picking the gatewayFQDN node.

Reply:
Changed this to use `find` and look for hbase and solr keytabs.